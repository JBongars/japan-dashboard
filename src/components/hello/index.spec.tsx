
import React from 'react';
import { mount, ReactWrapper } from 'enzyme';
import * as hello from './index';

describe('#hello', (): void => {
  it('should match a snapshot', (): void => {
    const wrapper: ReactWrapper = mount<React.Component>(<hello.default name="Jack" />);
    expect(wrapper).toMatchSnapshot();
  })
})
