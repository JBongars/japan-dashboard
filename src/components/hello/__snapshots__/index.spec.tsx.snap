// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#hello should match a snapshot 1`] = `ReactWrapper {}`;
